packages/frontend/app/components/LiquidityDialog.tsx

```diff
@@ -1,19 +1,43 @@
 import { Dialog } from '@headlessui/react'
 import { Form } from '@remix-run/react'
+import { ChangeEvent, useState } from 'react'
```

Comment:
our prettier check failed here: https://github.com/interledger/rafiki/actions/runs/10097295226/job/27927389684?pr=2817

can either fix with running `pnpm format` or:

```suggestion
import type { ChangeEvent } from 'react'
import { useState } from 'react'
```

Reply:
Done 👌